### DISCUSSION TITLE
936 Feedback Discussion (removes AI nuke)

### DISCUSSION THREAD

**Discussion by YakumoChen** [Bubberstation/Bubberstation#950] [codebase discussion]
Feedback for https://github.com/Bubberstation/Bubberstation/pull/936

Burger blocked me in the PR to stifle my completely fair criticism and alternative to their PR without needing to respond to it (painted as "bullying" when they literally have a removal pr catchphrase they made up)

So here's a thread instead for fair discussion of a major removal

**Comment by destrucktoid** [comment]
He also blocked me as well
